src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/aggregation/aggregation.component.html

```diff
@@ -1,4 +1,10 @@
-<mat-expansion-panel class="aggregation mat-elevation-z0" [expanded]="canExpand" [disabled]="!active">
+<mat-expansion-panel
+  class="aggregation mat-elevation-z0"
+  [(expanded)]="expanded"
+  (click)="setShouldExpand()"
```

Comment:
Check if this logic can be implemented in another way instead of the click event if not consider the enter event also